src/react_native/reanimated.cljs

```diff
@@ -75,4 +82,11 @@
 
 (defn animate-shared-value-with-repeat [anim val duration easing number-of-repetitions reverse?]
   (set-shared-value anim (with-repeat (with-timing val (js-obj "duration" duration
-                                                               "easing"   (get easings easing))) number-of-repetitions reverse?)))
\ No newline at end of file
+                                                               "easing"   (get easings easing))) number-of-repetitions reverse?)))
+
+(defn animate-shared-value-with-delay-repeat
+  ([anim val duration easing delay number-of-repetitions]
+   (animate-shared-value-with-delay-repeat anim val duration easing delay number-of-repetitions false))
+  ([anim val duration easing delay number-of-repetitions reverse?]
+   (set-shared-value anim (with-delay delay (with-repeat (with-timing val (js-obj "duration" duration
```

Comment:
Could you reformat this piece differently? I find it really hard to read. It's common for functions or macros that start with `with-` to have the second argument in the next line. If you follow this idea you end up with something more readable in this particular code.

And how about using the `#js` literal instead of `js-obj`?

```clojure
(defn animate-shared-value-with-delay-repeat
  ([anim val duration easing delay number-of-repetitions]
   (animate-shared-value-with-delay-repeat anim val duration easing delay number-of-repetitions false))
  ([anim val duration easing delay number-of-repetitions reverse?]
   (set-shared-value anim
                     (with-delay delay
                       (with-repeat
                         (with-timing val
                           #js {:duration duration
                                :easing   (get easings easing)})
                         number-of-repetitions reverse?)))))
```

Reply:
Done. Probably other functions in this file need to be reformatted, as I just copied how the other ones were implemented. Probably will raise a PR with those refactors after this one.